kolena/dataset/evaluation.py

```diff
@@ -215,10 +217,12 @@ def _upload_results(
     load_uuid, dataset_id, total_rows = _prepare_upload_results_request(dataset, model, results)
 
     response = _send_upload_results_request(model, load_uuid, dataset_id, sources=sources)
+    models = serialize_models_url(response.model_id, response.eval_config_id)
     log.info(
         f"uploaded test results for model '{model}' on dataset '{dataset}': "
         f"{total_rows} uploaded, {response.n_inserted} inserted, {response.n_updated} updated",
     )
+    log.info(f"{get_platform_url()}/dataset/studio?datasetId={dataset_id}&models={models}")
```

Comment:
Let's follow the same pattern we use in [`upload_dataset`](https://github.com/kolenaIO/kolena/blob/trunk/kolena/dataset/dataset.py#L251) and append the URL to the end of the previous log line, wrapped by parentheses.

Reply:
For the case of multi config on the same upload should we show a list: 

```
uploaded test results for model 'model_XYZ' on dataset 'dataset_123': 100 uploaded, 95 inserted, 5 updated.
Model Link for Config {'threshold': 0.1, ...}: 
 - http://example.com/dataset/standards?datasetId=1&models=model=01...
Model Link for Config {'threshold': 0.5, ...}: 
 - http://example.com/dataset/standards?datasetId=1&models=model=02...
Model Link for Config {'threshold': '09', ...}: 
 - http://example.com/dataset/standards?datasetId=1&models=model=03...

```

Reply:
Updated.